tsconfig.json

```diff
@@ -1,13 +1,20 @@
 {
+  "extends": "./node_modules/@dojo/scripts/tsconfig/commonjs.json",
 	"compilerOptions": {
 		"declaration": false,
+		"lib": [
+			"dom",
```

Comment:
if our target is `es2016` do we need to enumerate the libs or can we just use `es2016`? Also, where are we using `dom`?

Reply:
`@dojo/core` (used by Intern) uses DOM definitions. For some reason, I've had to enumerate libs in all of the CLI projects even when specifying the target.